src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  }
+
+  if (days === 3) {
+    return 100;
```

Comment:
The return value for exactly 3 days is hard-coded to 100, which should be calculated based on the daily rate and the appropriate discount. Use constants and calculations instead of hard-coded values.